modules/porous_flow/include/materials/PorousFlowMaterialCapillaryPressureVG.h

```diff
@@ -0,0 +1,57 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+
+#ifndef POROUSFLOWMATERIALCAPILLARYPRESSUREVG_H
+#define POROUSFLOWMATERIALCAPILLARYPRESSUREVG_H
+
+#include "PorousFlowMaterialCapillaryPressureBase.h"
+
+//Forward Declarations
+class PorousFlowMaterialCapillaryPressureVG;
+
+template<>
+InputParameters validParams<PorousFlowMaterialCapillaryPressureVG>();
+
+/**
+ * van Genuchten capillary pressure for multiphase flow in porous media.
+ *
+ * Based on van Genuchten, M. Th., A closed for equation for
+ * predicting the hydraulic conductivity of unsaturated soils,
+ * Soil Sci. Soc., 44, 892-898 (1980).
+ */
+class PorousFlowMaterialCapillaryPressureVG : public PorousFlowMaterialCapillaryPressureBase
+{
+public:
+  PorousFlowMaterialCapillaryPressureVG(const InputParameters & parameters);
+
+protected:
+
+  virtual void computeQpProperties();
+
+  /// Capillary pressure as a function of phase saturation
+  Real capillaryPressure(Real saturation) const;
+  /// Derivative of capillary pressure wrt phase saturation
+  Real dCapillaryPressure(Real saturation) const;
+  /// Second derivative of capillary pressure wrt phase saturation
+  Real d2CapillaryPressure(Real saturation) const;
```

Comment:
You sprinkle empty lines where they don't belong and then lump lines together where a few empties would really improve readability. I'd replace the 6 lines above with 

```
  ///@{ Capillary pressure as a function of phase saturation and its derivatives
  Real capillaryPressure(Real saturation) const;
  Real dCapillaryPressure(Real saturation) const;
  Real d2CapillaryPressure(Real saturation) const;
  ///@}

```

This visually groups the methods and avoids redundant comments.